src/work-allocation/components/case-reference-field/case-reference-field.component.spec.ts

```diff
@@ -0,0 +1,93 @@
+import { Component, Input, ViewChild } from '@angular/core';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { WorkAllocationComponentsModule } from '../work-allocation.components.module';
+import { CaseReferenceFieldComponent } from './case-reference-field.component';
+
+@Component({
+  template: `<exui-case-reference-field [caseReference]="caseReference"></exui-case-reference-field>`
+})
+class WrapperComponent {
+  @ViewChild(CaseReferenceFieldComponent) public appComponentRef: CaseReferenceFieldComponent;
+  @Input() public caseReference: string;
+}
+
+describe('WorkAllocation', () => {
+
+  describe('CaseReferenceFieldComponent', () => {
+    let component: CaseReferenceFieldComponent;
+    let wrapper: WrapperComponent;
+    let fixture: ComponentFixture<WrapperComponent>;
+
+    beforeEach(async(() => {
+      TestBed.configureTestingModule({
+        declarations: [ WrapperComponent ],
+        imports: [ WorkAllocationComponentsModule ]
+      })
+      .compileComponents();
+    }));
+
+    beforeEach(() => {
+      fixture = TestBed.createComponent(WrapperComponent);
+      wrapper = fixture.componentInstance;
+      component = wrapper.appComponentRef;
+      fixture.detectChanges();
+    });
+
+    it('should handle a CASE_REFERENCE type', () => {
```

Comment:
Is there a reason all the checks are in one single test instead of separate tests?

Reply:
Fixed.